azure-pipelines.yml

```
@@ -112,7 +120,7 @@ steps:
   condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
   inputs:
     SourcePath: '$(shutter)'
-    azureSubscription: 'DCD-CFTAPPS-PROD'
+    azureSubscription: ${{ variables.subscription }}
```

use consistent syntax
```suggestion
    azureSubscription: $(subscription)
```

---

I'm pretty sure I tried that and it failed :( will try it though